py_neuromodulation/nm_sharpwaves.py

```diff
@@ -256,8 +263,8 @@ def analyze_waveform(self) -> None:
 
         """ Find left and right peak indexes for each trough """
         peak_pointer = 0
-        peak_idx_left = []
-        peak_idx_right = []
+        peak_idx_left_list = []
```

Comment:
That's great! More intuitive to know it's a list

Reply:
This was mostly to stop MyPy from complaining about redefining variables, but since I'm used to C++ where you just can't never modify type for me it also makes sense, it helps keep tracking of the manipulations. Also if that JIT compiler thing with Python ends up being relevant (or in the event we can make PyPy work with the project) having distinct variables by type proably helps too.